### ISSUE TITLE
Suffix can't be empty in the CLI

### ISSUE THREAD

**Issue by dariusrosendahl** [dohomi/storyblok-generate-ts#47]
Hi there! 

First off. Thanks for this package! Makes working with SB a lot easier!

I'm having the following problem: perhaps I'm using it wrong, but I can't seem to generate interface names without a suffix.  

```
//package.json
		"generate-sb-types": "storyblok-generate-ts source=./components.XXXXXX.json target=./component-types-sb titlePrefix='I' titleSuffix=''",
```

Expected result:
```
export interface Ifeature {
  name?: string;
  _uid: string;
  component: "feature";
  [k: string]: any;
}
```

Actual result:
```
export interface IfeatureStoryblok {
  name?: string;
  _uid: string;
  component: "feature";
  [k: string]: any;
}
```

**Comment by dohomi** [comment]
I need to look into it but currently on total different mission at work. will try to spare some time on the weekend

**Comment by dariusrosendahl** [comment]
Anything I can do to help merge #52 and get this into production @dohomi ?

**Comment by dohomi** [comment]
released with `1.15.1`
